mod/01_rating/set/type_plus_right/metric/all_metric_values.rb

```diff
@@ -1,4 +1,4 @@
-include_set Abstract::AllValues
+include_set Abstract::AllMetricValues
```

Comment:
I renamed it but now we use that abstract set also for lists of companies and metrics. We probably should come up with a better name